src/test/bench/testfilestore_backend.cc

```diff
@@ -57,9 +57,10 @@ void TestFileStoreBackend::write(
 
   os->queue_transaction(
     osr,
-    t,
+    std::move(*t),
```

Comment:
This one looks a bit fishy. Making the rvalue reference is a promise that no other references exist, which is not the case here: we pass a pointer to C_DeleteTransWrapper which is going to delete it (depending on luck the object might be already deleted)